guava/src/com/google/common/util/concurrent/RateLimiter.java

```diff
@@ -421,6 +421,18 @@ public boolean tryAcquire(int permits, long timeout, TimeUnit unit) {
     return true;
   }
 
+  public long microsToNextPermit() {
+    return queryEarliestAvailable(stopwatch.readMicros());
+  }
+
+  public boolean canAcquire() {
+    return canAcquire(stopwatch.readMicros(), 0);
+  }
+
+  public boolean canAcquire(long timeoutMicros) {
```

Comment:
Given Google's recent "GoodTime" effort I expect that this method should be replaced with `canAcquire(Duration timeout)`.

Reply:
Finally got around to updating the PR with your suggestions. 😅